src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInternalTransaction.cs

```diff
@@ -17,7 +17,7 @@ internal enum TransactionState
         Aborted = 2,
         Committed = 3,
         Unknown = 4,
-    }
+    };
```

Comment:
This is an enum type declaration so the `;` is not needed here or the one above, please remove them.